dts/bindings/i2c/nordic,nrf-twi-common.yaml

```diff
@@ -16,9 +16,22 @@ properties:
     sda-pin:
       type: int
       required: true
-      description: SDA pin
+      description: |
+        The SDA pin to use.
+
+        For pins P0.0 through P0.31, use the pin number. For example,
+        to use P0.16 for SDA, set:
+
+            sda-pin = <16>;
+
+        For pins P1.0 through P1.31, add 32 to the pin number. For
+        example, to use P1.2 for SDA, set:
+
+            sda-pin = <34>;  /* 32 + 2 */
```

Comment:
It really would be nice to be able to use `<&gpio0 16 0>` and `<&gpio1 2 0>` like every other vendor, and `NORDIC_DT_GPIO_ORDINAL(node_id, prop)` to construct the value.  I'm afraid it's too late, though.

Reply:
> It really would be nice to be able to use `<&gpio0 16 0>` and `<&gpio1 2 0>` like every other vendor, and `NORDIC_DT_GPIO_ORDINAL(node_id, prop)` to construct the value. I'm afraid it's too late, though.

I want to start by documenting what we have. Now that we have the ability to deprecate properties, we can introduce alternatives and gracefully deprecate these things over time.